src/components/ebay-window-notice/examples/02-window-notice-fill/template.marko

```diff
@@ -0,0 +1,17 @@
+<style>
+    .demo2-container {
+        max-width: 320px;
+        height: 500px;
+        padding: 8px;
+        background: #fff;
+        border: 1px solid #eee;
+    }
+</style>
+
+<div class="demo2-container">
+    <ebay-window-notice type="window" a11y-text="Success!" fill-window>
```

Comment:
I think I'd prefer an enumerated type here instead of the `fill-window` boolean. I don't see this as something that would get toggled on and off like other booleans (e.g. disabled, expanded, hidden). Perhaps we could make that part of the criteria for a boolean or not? @DylanPiercey @agliga thoughts?

Reply:
I would personally not have enums unless they have 2+ options or will possibly have more options in future. The reason is because there will only be 1 option you will specify. Like: `fill="window"` or `"fill="none"` where you'll never want to use `fill="none"`. I think it becomes a waste rather than doing `fill-window`. But that's just me.

Reply:
Right now there are only two, but it's not inconceivable that more could be added, e.g. `layout="default"`, `layout="window"`, `layout="split".

Maybe just me, but I don't have a huge problem with only 2 options. There are cases of that in html, css and aria too. 

I think we should try and set some firm rules for when to use a boolean. I guess for me I'm more used to seeing booleans used for things that are semantic rather than presentational.

Reply:
I would be interested to see if we could follow this "rule" with our API in v6, but open to more discussion...

Reply:
@agliga To unblock this, let's move forward with this stricter guideline on use of booleans and change this to be an enum.

The attribute name and values should reflect the two current use cases. Given that window notices appear only in dialog windows, my suggestion is to go with `window=lightbox` (default) and `window=fullscreen` for now. It is not unfeasible that we may add a `window=panel` or `window=drawer` in future, and this would tie in with our new updated dialog terminology.

![Screen Shot 2020-07-27 at 9 46 49 PM](https://user-images.githubusercontent.com/38065/88620373-b9b89980-d052-11ea-94ab-4532c21d49c0.png)